.github/workflows/hosted.yml

```diff
@@ -166,16 +166,17 @@ jobs:
         ]
         target: [
           tool,
-          cpp,
-          csharp,
-          dart,
-          go,
-          java,
-          javascript,
-          typescript,
-          php,
-          python3,
+          # cpp,
+          # csharp,
+          # dart,
+          # go,
+          # java,
+          # javascript,
+          # typescript,
+          # php,
+          # python3,
```

Comment:
I'll uncomment it before the final merge.

I commented it out for more faster remote building when I was trying to fix Windows CI.